wormhole-connect/src/hooks/useRoutesQuotesBulk.ts

```diff
@@ -24,6 +27,16 @@ const useRoutesQuotesBulk = (routes: string[], params: Params): HookReturn => {
   const [isFetching, setIsFetching] = useState(false);
   const [quotes, setQuotes] = useState<QuoteResult[]>([]);
 
+  // TODO temporary
+  // Calculate USD amount for temporary $1000 Mayan limit
```

Comment:
I think this comment belongs to line 85 below?

Reply:
it belongs in both places. I'll copy it